### PR TITLE
tickets/DM 39324

### PR DESCRIPTION
**Pull request by couger01**
- Add license headers
- Add tcpip client

### CODE REVIEW

File: python/lsst/ts/cbp/mock_server.py [hunk 1]
```diff
@@ -87,7 +107,7 @@ def __init__(self, log=None):
         self.park = False
         self.auto_park = False
         self.masks_rotation = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
-        self.terminator = "\r\n"
+        terminator = b"\r\n"
```
Comment (wvreeven) [inline python/lsst/ts/cbp/mock_server.py:110]: Please use `tcpip.DEFAULT_TERMINATOR` instead. Even better, remove this. See my next comment.

File: python/lsst/ts/cbp/mock_server.py [hunk 2]
```diff
@@ -120,16 +140,11 @@ def __init__(self, log=None):
         super().__init__(
             name="CBP Mock Server",
             host=tcpip.LOCAL_HOST,
-            port=9999,
-            connect_callback=self.connect_callback,
+            port=0,
             log=self.log,
+            terminator=terminator,
```
Comment (wvreeven) [inline python/lsst/ts/cbp/mock_server.py:145]: This is not necessary. The default value for `terminator` is `tcpip.DEFAULT_TERMINATOR` which has exactly the same value of `b"\r\n"`.

**Review by wvreeven — CHANGES_REQUESTED** [review]
I am requesting changes since you didn't update the version history. Please do that and process my comments as well and then re-request my review.

File: conda/meta.yaml [hunk 3]
```diff
@@ -25,17 +24,14 @@ test:
         - ts-dds
         - ts-idl {{ idl_version }}
         - ts-salobj {{ salobj_version }}
-        - ts-utils
-        - ts-tcpip
-        - ts-simactuators
```
[the remark this answers is not in the document]
Comment (couger01) [reply conda/meta.yaml:30]: Yep, checked it and removed from test section as redundant dependencies from host and runtime sections.

**Pull request merged**